### ISSUE TITLE
Wanted: co-maintainer

### ISSUE THREAD

**Issue by jcje-otg** [sphinx-contrib/images#19]
TL; DR; A second maintainer would be welcome, if you're interested let me know. Until summer 2021 I have limited time to dedicated to merging PRs  and handling issues.

I only intended to help complete https://github.com/spinus/sphinxcontrib-images/pull/56 by leaving a comment, to gain Sphinx 2.x support which I needed at the time, but to move forward the project ended up under the https://github.com/sphinx-contrib umbrella with me as admin (on https://pypi.org/project/sphinxcontrib-images/ as well).

There are some fine issues waiting to be fixed and some fine looking PRs to fix some of the issues, but I have had very little time to devote to the project since May 2020 and I will continue to have little time until late spring or early summer 2021.

Adding a second maintainer or co-maintainer could help move the project forward at a more steady pace. Let me know if you're interested. Tasks would include one or more of answering issue tickets, reviewing PRs, merging PRs, releasing to PyPi. I'm no expert in any of this, but I'll be happy to help get new co-maintainers started on the tasks.

**Comment by jonascj** [comment]
Wrong account *sigh*
